### PR TITLE
Add homeassistant

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
Waiting for https://github.com/conda-forge/staged-recipes/pull/16272 and https://github.com/conda-forge/astral-feedstock/pull/9

**Comment by jan-janssen** [comment]
Include license in pip package upstream https://github.com/home-assistant-libs/voluptuous-serialize/pull/32

**Comment by jan-janssen** [comment]
The majority of the changes in the patch are already included in the latest development version. I just created a pull request to support the latest pip version upstream https://github.com/home-assistant/core/pull/56674

**Comment by jan-janssen** [comment]
```
WARNING: failed to get_linkages, codefile_type('C:\Miniconda\conda-bld\homeassistant_1632635709959\_h_env\Scripts\hass.exe')=EXEfile
   INFO: sysroot: 'C:/Windows/' files: '['winhlp32.exe', 'win.ini', 'twain_32/wiatwain.ds', 'twain_32.dll']'
WARNING (homeassistant): interpreted library (Python) package conda-forge::pip-21.2.4-pyhd8ed1ab_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
   INFO (homeassistant): Interpreted package 'homeassistant' is interpreted by 'python'
[noarch_python] Noarch package contains binary script: hass.exe
```

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/homeassistant/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/homeassistant-{{ version }}.tar.gz
+  sha256: da587c9fbdfd41169aeb4802d1458dac81a46f026644c29b29b5a8cbcdcb7719
+  patches:
+    - pyjwt.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<38]
+  skip: true  # [py>38]
```
Comment (ocefpaf) [inline recipes/homeassistant/meta.yaml:19]: Why not `#  [not py38]` instead?

**Pull request merged**